impl/src/templates/remote_crates.bzl.template

```diff
@@ -10,7 +10,10 @@ def {{workspace.gen_workspace_prefix}}_fetch_remote_crates():
         name = "{{workspace.gen_workspace_prefix}}__{{crate.pkg_name | slugify | replace(from="-", to="_")}}__{{crate.pkg_version | slugify | replace(from="-", to="_")}}",
         url = "https://crates-io.s3-us-west-1.amazonaws.com/crates/{{crate.pkg_name}}/{{crate.pkg_name}}-{{crate.pkg_version}}.crate",
         type = "tar.gz",
-        strip_prefix = "{{crate.pkg_name}}-{{crate.pkg_version}}",
+        {% if crate.sha256
```

Comment:
Nit: use "{%-" to accomplish the same thing as you're accomplishing by splitting the tag.

Example: https://github.com/google/cargo-raze/blob/master/impl/src/templates/crate.BUILD.template#L23

Reply:
Much nicer. Done!